packages/plans-grid/src/plans-interval-toggle/index.tsx

```diff
@@ -0,0 +1,95 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { useI18n } from '@automattic/react-i18n';
+import { sprintf } from '@wordpress/i18n';
+import { Popover } from '@wordpress/components';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import SegmentedControl from '../segmented-control';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export type BillingIntervalType = 'yearly' | 'monthly';
+
+type PopupMessageProps = {
+	isVisible: boolean;
+};
+
+export const PopupMessages: React.FunctionComponent< PopupMessageProps > = ( {
+	children,
+	// isVisible,
+} ) => {
+	const positions: React.ComponentProps< typeof Popover >[ 'position' ][] = [
+		'middle right',
+		'bottom center',
+	];
+
+	return (
+		<>
+			{ positions.map( ( pos ) => (
+				<Popover key={ pos } className="plans-interval-toggle__popover" position={ pos }>
+					{ children }
+				</Popover>
+			) ) }
+		</>
+	);
+};
+
+type ToggleHostProps = {
+	intervalType: BillingIntervalType;
+	onChange: ( selectedValue: BillingIntervalType ) => void;
+	maxSavingsPerc: number;
+};
+
+const PlansIntervalToggle: React.FunctionComponent< ToggleHostProps > = ( {
+	onChange,
+	intervalType,
+	maxSavingsPerc,
+} ) => {
+	const { __ } = useI18n();
+
+	const annuallySpanRef = React.useRef( null );
+
+	const segmentClasses = classNames( 'plan-features__interval-type', 'price-toggle', {
+		'is-monthly-selected': intervalType === 'monthly',
+	} );
+
+	return (
+		<SegmentedControl compact className={ segmentClasses } primary={ true }>
+			<SegmentedControl.Item
+				selected={ intervalType === 'monthly' }
+				onClick={ () => onChange( 'monthly' ) }
+			>
+				<span>{ __( 'Monthly', __i18n_text_domain__ ) }</span>
+			</SegmentedControl.Item>
+
+			<SegmentedControl.Item
+				selected={ intervalType === 'yearly' }
+				onClick={ () => onChange( 'yearly' ) }
+			>
+				<span ref={ annuallySpanRef }>{ __( 'Annually', __i18n_text_domain__ ) }</span>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Pay annually' )` *ES Score: 8*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/feat/plans-grid-billing-period-toggle)